features/project/project_controller.rb

```diff
@@ -18,6 +18,13 @@ class ProjectController < AuthenticatedControllerBase
 
       repo = FastlaneCI::GitRepo.new(git_config: project.repo_config, provider_credential: current_github_provider_credential)
       current_sha = repo.most_recent_commit.sha
+      manual_triggers_allowed = project.job_triggers.any? { |trigger| trigger.type == FastlaneCI::JobTrigger::TRIGGER_TYPE[:manual] }
+
+      unless manual_triggers_allowed
+        status(403) # Forbidden
```

Comment:
I don't think we want to return any statuses, but rather, use the notification pipeline to send the message. For right now, this is totally OK, though.

Reply:
Yeah, good point, I guess this all very much depends on what we decide on for the frontend

Reply:
I think we should continue to explore single page webapps, but still go full steam ahead with our current design for MVP. Remembering that the M in MVP is Minimum 😄 
It would be trivial to migrate off ERBs and into a single-page webapp. I just don't want to do that right now because then we'd be committing to building a `client` on top of the server, and having to handle all the complexities that come along with that.

Reply:
@taquitos could you add that to the web app issue ^